polars/polars-ops/src/series/ops/rolling.rs

```diff
@@ -59,6 +85,44 @@ pub trait RollingSeries: SeriesSealed {
             )),
         }
     }
+
+    fn rolling_kurtosis(
+        &self,
+        window_size: usize,
+        fisher: bool,
+        bias: bool,
+    ) -> PolarsResult<Series> {
+        let s = self.as_series();
+
+        if window_size < 4 {
+            Err(PolarsError::ComputeError(format!(
+                "cannot use rolling_kurtosis with window_size < 4, window_size is: {}",
+                window_size
+            )))
+        }
```

Comment:
This seems overly restrictive. Does `pandas` require `window_size >= 4` also?

Reply:
https://pandas.pydata.org/docs/reference/api/pandas.core.window.rolling.Rolling.kurt.html

According to the notes, 4 periods are required. So I guess we'd need to doc that